engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala

```diff
@@ -81,7 +81,8 @@ class DeploymentPreparer(config: K8sDeploymentManagerConfig) extends LazyLogging
       val containerLens = GenLens[Container](_.env).modify(_ ++ runtimeContainer.env) andThen
         GenLens[Container](_.volumeMounts).modify(_ ++ runtimeContainer.volumeMounts) andThen
         GenLens[Container](_.readinessProbe).modify(_.orElse(runtimeContainer.readinessProbe)) andThen
-        GenLens[Container](_.livenessProbe).modify(_.orElse(runtimeContainer.livenessProbe))
+        GenLens[Container](_.livenessProbe).modify(_.orElse(runtimeContainer.livenessProbe)) andThen
+        GenLens[Container](_.image).set(runtimeContainer.image)
```

Comment:
If someone will use other image in config, he/she can be confused that it was overridden. Maybe we can change it to:
```
.modify { configuredImage =>
  if (configuredImage != runtiemContainer.image) logger.warn...
  runtiemContainer.image
}
```
WDYT?

Reply:
or even throw exception if image is set :)

Reply:
@mproch @arkadius would do we prefer ? log.warn and throwing exception looks as if two alternative approaches ? @arkadius in case one do you think we should also log such a warning for other configs as label selector etc ?

Reply:
@dswiecki which one you prefer? I agree that throwing is also fine, probable even better. Regards to other places where `set` was used IMO we can behave the same. If we find out cases where it disallow some configuration, we will think how to handle it.